source/functions/handle-errors.txt

```diff
@@ -0,0 +1,422 @@
+               :caption: triggers/retryOperation.json
+
+      Now add the code for the Function that the Trigger invokes.
+
+      The function ``retryOperation`` takes as a parameter ``logEntry``, the document that the
+      retry handler posted to the failed execution tracker collection.
+      Then ``retryOperation`` use :ref:`context.functions.execute() <context-functions>`
+      to invoke the main function with information from ``logEntry``.
+
+      .. tabs-realm-admin-interfaces::
+         :hidden: true
+
+         .. tab::
+            :tabid: ui
+
+            In the field :guilabel:`Function Name`, add ``retryOperationDbTrigger``.
+
+            For the field :guilabel:`Function`, add the following code:
+
+            .. literalinclude:: /examples/generated/functions/retryOperationDbTrigger.snippet.retry-operation-db-trigger-function.js
+               :language: js
+               :caption: functions/retryOperationDbTrigger.js
+
+         .. tab::
+            :tabid: cli
+
+            Add the Function metadata to :file:`functions/config.json`:
+
+            .. code-block:: js
+               :caption: functions/config.json
+
+               [
+                  // ...other configuration
+                  {
+                    "name": "retryOperationDbTrigger",
+                    "private": true
+                  }
+               ]
+
+            Add the following code to the file :file:`functions/retryOperationDbTrigger.js`:
+
+            .. literalinclude:: /examples/generated/functions/retryOperationDbTrigger.snippet.retry-operation-db-trigger-function.js
+               :language: js
+               :caption: retryOperationDbTrigger.js
+
+   .. step:: Write Function to retry
+
+      Now that you have the function handler and the retry Database Trigger Function,
+      you can write the main function.
+
+      In this example, the Function randomly throws an error when performing addition.
+      The JavaScript functions that execute this logic are the following:
+
+      - ``getRandomOneTwoThree()``: Helper function for generating errors
+        for the example.
+      - ``additionOrFailure()``: Function with the main logic.
+
+      The invocation of ``additionOrFailure()`` wrapped by the retry handler
+      occurs in the exported function ``additionWithRetryHandler()``.
+      **All** functions that use the retry handler function should resemble this function.
+
+      You must include the correct parameters to make this function work
+      with the rest of the retry logic. These parameters are:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 50 50
+
+         * - Parameter
+           - Description
+
+         * - ``...args``
+           - Zero or more parameters to pass to the function with main logic.
+             In the case of this example, the two numbers added
+             in ``additionOrFailure()``, ``num1`` and ``num2``.
+
+         * - ``operationId``
```

Comment:
In this table and the one below, I would probably break out the type information into a separate column.